### DISCUSSION TITLE
Title misalginment in reports

### DISCUSSION THREAD

**Discussion by jeff282** [burkybang/SuiteAdvanced-Feedback#80] [Ideas]
I don't know if it's just something in my browser or not but i notice with SA the report titles are misaligned. When i disable SA it appears correctly. 
ON:
![CleanShot 2023-04-05 at 23 19 11@2x](https://user-images.githubusercontent.com/130023634/230266051-3d9ae7eb-1671-408b-be75-dd708319f831.png)

Off:
![CleanShot 2023-04-05 at 23 37 52@2x](https://user-images.githubusercontent.com/130023634/230266310-6f49c383-30c2-4495-8816-a0a51c96f153.png)

**Comment by burkybang** [comment]
It's happening for me too. Great catch! I'll get it fixed.

**Comment by burkybang** [comment]
@jeff282 I just checked this again, and it doesn't seem to be happening for me anymore. I don't remember fixing it. Is it still happening for you?

**Reply by jeff282** [reply]
Just checked - looks like it is good now! Thanks!
